### DISCUSSION TITLE
Privacy setup

### DISCUSSION THREAD

**Discussion by visskiss** [mempool/mempool#1368] [Q&A]
Can someone assist with the correct config to route all outgoing data calls over tor (for price for example)? Is it enough to setup the proxy and then the tor price URL?  How can we be sure that these requests are taking place over tor?  

Thanks

**Comment by wiz** [comment]
The only "outgoing data call" is for the price feed, which can be routed over Tor by setting the proxy variables. It is clearly indicated if using clearnet or Tor in the logs. All other data is obtained from your local bitcoind and (if configured) your electrum or esplora backend.

**Comment by ghost** [comment]
@visskiss I am currently miltigating this issue #1649, #1651, #1691.

If this issue is fully resolved will be leaving updates here.
